src/index.html

```diff
@@ -15,8 +15,50 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="http://localhost:8080/">
+          <img
+            src="./images/logo.png"
+            alt="Moyo_shop_Gadgets"
+            width="40px"
+            height="40px"
+          />
+        </a>
+      </div>
+      <nav aria-label="Main navigation">
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li><a href="#">SAMSUNG</a></li>
+          <li><a href="#">SMARTPHONES</a></li>
+          <li>
+            <a
+              href="#"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li><a href="#">GADGETS</a></li>
+          <li><a href="#">TABLETS</a></li>
+          <li><a href="#">PHOTO</a></li>
+          <li><a href="#">VIDEO</a></li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
Consider adding more ARIA attributes or roles to improve accessibility, especially for the navigation menu. This can help users with assistive technologies better understand the structure and purpose of the navigation elements.